File: java/src/main/java/com/example/ordermgmt/model/OrderInput.java
package com.example.ordermgmt.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class OrderInput {
    @JsonProperty("OrderId")
    private String orderId;

    @JsonProperty("Address")
    private String address;
}

File: dotnet/src/model/OrderInput.cs
using System.Text.Json.Serialization;

namespace DotNetOrderManagement.model;

public class OrderInput
{
    [JsonPropertyName("OrderId")]
    public string OrderId { get; set; }
    [JsonPropertyName("Address")]
    public string Address { get; set; }

    public OrderInput()
    {
        OrderId = string.Empty;
        Address = string.Empty;
    }

    public OrderInput(string orderId, string address)
    {
        this.OrderId = orderId;
        this.Address = address;
    }
}